web/config/options.js

```diff
@@ -41,6 +46,12 @@ module.exports = {
         payload: {
           maxBytes: 5242880,
         },
+        cors: {
+          origin: [hostWithPort, 'https://changex.org', 'https://www.changex.org', 'https://coderdojo.com', `http://localhost:${port}`],
```

Comment:
There's no need for our own host to appear in the CORS header, the browser won't care because it's not cross origin.